Microsoft.AVS.Management/Microsoft.AVS.Management.psd1

```diff
@@ -52,7 +52,7 @@
 
     # Modules that must be imported into the global environment prior to importing this module
     RequiredModules   = @(
-        @{"ModuleName" = "VMware.vSphere.SsoAdmin"; "RequiredVersion" = "1.3.8" },
+        @{"ModuleName" = "VMware.vSphere.SsoAdmin"; "RequiredVersion" = "1.3.5" },
```

Comment:
They release 1.3.9 with our fix apparently!
https://github.com/vmware/PowerCLI-Example-Scripts/pull/604#issuecomment-1456311113

Reply:
This feature doesn't need to upgrade the sso module. I will open another pr for the update-password feature.

Reply:
It doesn't need to downgrade it either, does it?